cfme/tests/infrastructure/test_config_management.py

```diff
@@ -101,34 +81,23 @@ def test_config_manager_add_invalid_url(request, config_manager_obj):
                         '(typical production settings)'
 
     with error.expected(error_message):
-        if config_manager_obj.type == "Ansible Tower":
-            config_manager_obj.create(validate=False)
-        else:
-            config_manager_obj.create()
+        config_manager_obj.create()
 
 
 @pytest.mark.tier(3)
-@pytest.mark.uncollectif(lambda config_manager_obj: config_manager_obj.type == "Ansible Tower" and
-    version.current_version() < "5.6")
-@pytest.mark.meta(blockers=[BZ(1319751, forced_streams=["5.5", "5.6"])])
 def test_config_manager_add_invalid_creds(request, config_manager_obj):
     request.addfinalizer(config_manager_obj.delete)
     config_manager_obj.credentials.principal = 'invalid_user'
     with error.expected('Invalid username/password'):
-        if config_manager_obj.type == "Ansible Tower":
-            config_manager_obj.create(validate=False)
-        else:
-            config_manager_obj.create()
+        config_manager_obj.create()
 
 
 @pytest.mark.tier(3)
-@pytest.mark.uncollectif(lambda config_manager_obj: config_manager_obj.type == "Ansible Tower" and
-    version.current_version() < "5.6")
-@pytest.mark.meta(blockers=[
-    BZ(
-        1326316,
-        unblock=lambda config_manager_obj: config_manager_obj.type == "Ansible Tower")
-])
+@pytest.mark.meta(
+    blockers=[BZ(1388928, unblock=lambda config_manager_obj: (
+        config_manager_obj.type == "Ansible Tower" and version.current_version() > "5.7.0.9") or
```

Comment:
this might read better if it was written as `version.current_version() > "5.7.0.9" or config_manager_obj.type != "Ansible Tower"`